WORKSPACE.bazel

```diff
@@ -1 +1,12 @@
 # Marker that this is the root of a Bazel workspace.
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
Maybe I don't understand bazel, but should this be here? should examples have it's own workspace?

Reply:
unfortunately yes, module.bazel doesn't have an easy way to use http_archive in module.bazel files.

Reply:
this not necessary in bazel 7 but we are testing against 6 as well. so we are stuck using this